README.md

```diff
@@ -35,20 +35,17 @@ emulator or in a physical phone.
 
 # Testing
 
-In 492 you will write automated tests. When you do you will need to add a
-section that explains how to run them.
-
-The unit tests are in `/test/unit`.
-
-The behavioral tests are in `/test/casper/`.
+The unit tests use the regexep '*.test.ts'
```

Comment:
This isn't the right regex, I don't remember exactly what is but check the files.